src/main/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOperation.scala

```diff
@@ -39,10 +39,13 @@ object InstanceUpdateOperation {
 
   /**
     * @param instanceId Designating the instance that shall be launched.
-    * @param newTaskId The id of the task that will be launched via Mesos
+    * @param oldToNewTaskIds Mapping from old task IDs to new ones. It is required in order to maintain stable task IDs
+    *                        for resident tasks (currently, the ones, which have persistent volumes), which are launched
+    *                        upon a resource reservation. Stable here means that only the UUID and attempt part change,
```

Comment:
Only attemps changes, not the UUID part.

Reply:
Done.